maven-setup/src/main.ts

```diff
@@ -14,13 +15,16 @@ const MAVEN_OPTS_EXT_CLASS_PATH = '-Dmaven.ext.class.path'
 /**
  * Main entrypoint for the action
  */
-export function run(): void {
+export async function run(): Promise<void> {
```

Comment:
🤔 Just realized that we should not need to call `run()` line 58, GH should execute it when the action is called (to be tested though)

Reply:
I will test it and remove it before merging if not necessary 👍

Reply:
It works in regular case but when called with a relative path like in [the integration tests](https://github.com/gradle/develocity-actions/blob/a59b6b19681b5e6649fa7a22bf0001c90db0a769/.github/workflows/build.yml#L102), the action is not executed.

I will therefore not include this change.